packages/multicast_dns/pubspec.yaml

```diff
@@ -1,14 +1,10 @@
 name: multicast_dns
 description: Dart package for mDNS queries (e.g. Bonjour, Avahi).
 homepage: https://github.com/flutter/packages/tree/master/packages/multicast_dns
-version: 0.2.2
-
-dependencies:
-  meta: ^1.1.6
```

Comment:
Meta was only used for required and for a protected annotation, but that annotation was used on private members and I don't think its worth the dependency.